core/Plugin.php

```diff
@@ -451,6 +451,19 @@ public static function getPluginNameFromNamespace($namespaceOrClassName)
     }
 
     /**
+     * Override this method in your plugin class if you want your plugin to be loaded during tracking.
+     *
+     * Note: If you define your own dimension or handle a tracker event, your plugin will automatically
+     * be detected as a tracker plugin.
+     *
+     * @return bool
+     */
+    public function isTrackerPlugin()
```

Comment:
Re auto detecting maybe my question was rather would it be possible to put it in a `Tracker/RequestProcessors/` directory of a plugin and auto detect?